src/Toolkit/Toolkit/UI/Controls/BookmarksView/BookmarksViewDataSource.cs

```diff
@@ -268,46 +268,46 @@ private void RunOnUIThread(Action action)
 #if XAMARIN_FORMS
             global::Xamarin.Forms.Device.BeginInvokeOnMainThread(action);
 #elif __IOS__
-            _geoView.InvokeOnMainThread(action);
+            _geoView?.InvokeOnMainThread(action);
 #elif __ANDROID__
-            _geoView.PostDelayed(action, 500);
+            _geoView?.PostDelayed(action, 500);
 #elif NETFX_CORE
-            _ = _geoView.Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Low, () => action());
+            _ = _geoView?.Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Low, () => action());
 #else
-            _geoView.Dispatcher.Invoke(action);
+            _geoView?.Dispatcher.Invoke(action);
 #endif
         }
 
         #region IList<Bookmark> implementation
-        Bookmark IList<Bookmark>.this[int index] { get => ActiveBookmarkList?[index]; set => throw new NotImplementedException(); }
+        Bookmark IList<Bookmark>.this[int index] { get => ActiveBookmarkList[index]; set => throw new NotImplementedException(); }
 
-        int ICollection<Bookmark>.Count => ActiveBookmarkList?.Count ?? 0;
+        int ICollection<Bookmark>.Count => ActiveBookmarkList.Count;
 
         bool ICollection<Bookmark>.IsReadOnly => true;
 
         bool IList.IsReadOnly => true;
 
         bool IList.IsFixedSize => false;
 
-        int ICollection.Count => ActiveBookmarkList?.Count ?? 0;
+        int ICollection.Count => ActiveBookmarkList.Count;
 
         object ICollection.SyncRoot => throw new NotImplementedException();
 
         bool ICollection.IsSynchronized => false;
 
-        object IList.this[int index] { get => ActiveBookmarkList?[index]; set => throw new NotImplementedException(); }
+        object? IList.this[int index] { get => ActiveBookmarkList[index]; set => throw new NotImplementedException(); }
 
         void ICollection<Bookmark>.Add(Bookmark item) => throw new NotImplementedException();
 
         void ICollection<Bookmark>.Clear() => throw new NotImplementedException();
 
-        bool ICollection<Bookmark>.Contains(Bookmark item) => ActiveBookmarkList?.Contains(item) ?? false;
+        bool ICollection<Bookmark>.Contains(Bookmark item) => ActiveBookmarkList.Contains(item);
 
         void ICollection<Bookmark>.CopyTo(Bookmark[] array, int arrayIndex) => ActiveBookmarkList?.CopyTo(array, arrayIndex);
```

Comment:
I see some methods null-check ActiveBookmarkList and others don't. Let's be consistent. Is it nullable or no?